src/main/java/picard/analysis/AlignmentSummaryMetrics.java

```diff
@@ -122,13 +122,31 @@
      */
     public double PF_INDEL_RATE;
 
+    /** The median read length. Computed using all read lengths, not just the aligned ones. */
```

Comment:
Could you rephrase this second sentence to say: `Computed using all read lengths including clipped bases.` and include it in the other metrics you've changed below?

I think that's clearer and I'm basing that off of the difference between the two histograms that are included in AlignmentSummaryMetrics: https://github.com/broadinstitute/picard/blob/master/src/main/java/picard/analysis/AlignmentSummaryMetricsCollector.java#L332

Reply:
Made these changes in the following commit!